app/models/planet.py

```diff
@@ -0,0 +1,7 @@
+from app import db
+
+class Planet(db.Model):
+    id = db.Column(db.Integer, primary_key = True, autoincrement = True)
+    name = db.Column(db.String)
+    description = db.Column(db.String)
```

Comment:
Consider which columns we might want to make "required" (non-nullable). As it is, with this definition, we could make a Planet with a NULL name and description. Would we want to allow this to happen?